doc/bc/changes-8.0.md

```diff
@@ -61,6 +61,92 @@ Changes affecting version compatibility with former or future versions.
     * `eZ\Publish\Core\MVC\Symfony\Matcher\ContentMatcherFactory`
     * `eZ\Publish\Core\MVC\Symfony\Matcher\LocationMatcherFactory`
 
+* The following obsolete tables have been removed from the database schema:
+    - ezapprove_items,
+    - ezbasket,
+    - ezcollab_group,
+    - ezcollab_item,
+    - ezcollab_item_group_link,
+    - ezcollab_item_message_link,
+    - ezcollab_item_participant_link,
+    - ezcollab_item_status,
+    - ezcollab_notification_rule,
+    - ezcollab_profile,
+    - ezcollab_simple_message,
+    - ezcomment,
+    - ezcomment_notification,
+    - ezcomment_subscriber,
+    - ezcomment_subscription,
+    - ezcontentbrowserecent,
+    - ezcurrencydata,
+    - ezdiscountrule,
+    - ezdiscountsubrule,
+    - ezdiscountsubrule_value,
+    - ezenumobjectvalue,
+    - ezenumvalue,
+    - ezforgot_password,
+    - ezgeneral_digest_user_settings,
+    - ezinfocollection,
+    - ezinfocollection_attribute,
+    - ezisbn_group,
+    - ezisbn_group_range,
+    - ezisbn_registrant_range,
+    - ezm_block,
+    - ezm_pool,
+    - ezmessage,
+    - ezmodule_run,
+    - ezmultipricedata,
+    - eznotificationcollection,
+    - eznotificationcollection_item,
+    - eznotificationevent,
+    - ezoperation_memento,
+    - ezorder,
+    - ezorder_item,
+    - ezorder_nr_incr,
+    - ezorder_status,
+    - ezorder_status_history,
+    - ezpaymentobject,
+    - ezpdf_export,
+    - ezpending_actions,
+    - ezprest_authcode,
+    - ezprest_authorized_clients,
+    - ezprest_clients,
+    - ezprest_token,
+    - ezproductcategory,
+    - ezproductcollection,
+    - ezproductcollection_item,
+    - ezproductcollection_item_opt,
+    - ezpublishingqueueprocesses,
+    - ezrss_export,
+    - ezrss_export_item,
+    - ezrss_import,
+    - ezscheduled_script,
+    - ezsession,
+    - ezstarrating,
+    - ezstarrating_data,
```

Comment:
Rating FieldType

Reply:
> Rating FieldType

Either we need to drop entire FT or re-implement it, because it's not working since 1.x AFAIR (maybe w/ LegacyBridge it did). 

On the other hand, if there's an external package for this, compatible with eZ Platform 3.0, which I'm unaware of, then that package should create subscriber which will build those tables. I'm happy to provide that, just need to know where :)

Reply:
It's definitely not implemented in platform, never was. What is there is only a stub to make sure kernel does not bark at missing field type.

It would need a RatningService, Rating Criterion + SortClause and improvements to the field type itself.

So yes, you can opt to remove it also. Point is just table should go if FT goes right?

Reply:
> Point is just table should go if FT goes right?

Hmm, IMHO overkill, but ok, it's kind of separate scope. Restored `ezstarrating*` tables. I'll remove it separately.